### DISCUSSION TITLE
Image is not being saved

### DISCUSSION THREAD

**Discussion by gabzon** [spatie/laravel-medialibrary#2556] [Q&A]
Hi,

I just got the media library-pro. I'm trying to add a photo using a livewire component.

I have prepared my model as explained. and I followed this tutorial: https://spatie.be/docs/laravel-medialibrary/v9/handling-uploads-with-media-library-pro/handling-uploads-with-livewire

I've notice the table media and temporary_uploads are being populated but nothing is being saved when I check "storage/app/public"

`namespace App\Http\Livewire\Organization;

use App\Models\Organization;
use Livewire\Component;
use Spatie\MediaLibraryPro\Http\Livewire\Concerns\WithMedia;

class Form extends Component
{    
    use WithMedia;
    public Organization $organization;
    public $mediaComponentNames = ['logo'];
    public $logo;
    
    public function save()
    {
        $this->validate();

        $this->organization->save();
        
        $this->organization->addFromMediaLibraryRequest($this->logo)->toMediaCollection('organizations');
             
        session()->flash('success', 'Organization saved successfully');
        
        $this->clearMedia();
        
        return redirect()->route('organization.index');
    }`

I also notice I cannot use withResponsiveImages() method, is this a problem of addFromMediaLibraryRequest()?

I don't know if there is a bug or if I'm just missing something.

**Comment by freekmurze** [comment]
In our demo app (https://github.com/spatie/laravel-medialibrary-pro-app) you can see various examples on how you can use the media library pro components. Could you check if it works for you there?

**Comment by gabzon** [comment]
hi @freekmurze,

Thanks for your reply. I try to install the Laravel-medialibrary-pro-app on my Mac. I get this message when I run `composer intall` 

<img width="966" alt="Screenshot 2021-08-27 at 16 06 35" src="https://user-images.githubusercontent.com/142036/131140877-a381f9a4-8d04-45e5-8a81-bca37f369074.png">

It says `- spatie/pdf-to-image v2.x-dev requires php ^7.0 -> your php version (8.0.6) does not satisfy that requirement.`

Any ideas?
